import/llm/src/response.utils.ts

```
@@ -0,0 +1,54 @@
+  response: AIMessageChunk,
+  type: 'json' | 'javascript' | 'typescript' | 'markdown'
+): string | null => {
+  const content =
+    typeof response.content === 'string'
+      ? response.content
+      : JSON.stringify(response.content)
+
+  if (type === 'javascript') {
+    const javascriptRegex = new RegExp(
+      `\`\`\`javascript\\n([\\s\\S]*?)\\n\`\`\``
+    )
```

Biome

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)